### DISCUSSION TITLE
X Plane 12 crashes when I fly into a new area with Auroortho

### DISCUSSION THREAD

**Discussion by rayw45** [kubilus1/autoortho#257] [Q&A]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: [e.g. Windows 11]
- XPlane Version: [e.g. XP12 beta 4]
- AutoOrtho Version [e.g. 0.4.3]
- Filesystem type (NTFS, exFAT, etc)

**Additional context**
Add any other context about the problem here. (scenery version if applicable, etc, logfile snippet, etc)

### question

**Describe your question**
A clear and concise description of what the issue is.

**To Reproduce**
Steps to reproduce the behavior, if applicable:

**Screenshots**

**Comment by rayw45** [comment]
Windows 10, X Plane 12.05, AutoOrtho version 0.0.44_1 for Asia region, FIle system NTFS on fast SSD drives.

Every time since installing Auto Ortho that I fly into an area that I have not previously flown with AO I find X Plane crashes and I have to return to desktop.  The DOS window reports Empty files too. This also happens when I fly in Australia with the 0.0.42-1 aus version. 

My internet is very fast and reliable with a constant measures >900Mbps down and >700Mbps up and they do not block (well only some porn sites blocked here in Thailand by government order).  

My config file is set to ZL=16 and max wait I have now increased to 5.0 but still get crashes when flying into new areas. Is this maybe because the Bing site cannot download the tiles fast enough ??  I thought that if that is the case then it would still not crash X Plane but just give me blurry green default tiles ??

Anyway if anyone can help solve this frustrating problem as I love this fantastic program and project as it sir amazing especially working along with SimHeaven too

**Comment by dpdurst** [comment]
Duplicate the issue with a good explanation when it crashes and supply your Autoortho.log and Log.txt from XP. ZL=16 may be too much you should go back to defaults of 12 if this is the setting you changed in the config file below.

(#) minimum zoom level to allow.  this will not increase the max quality of satellite imagery
min_zoom = 12

If your flying Jets then indeed seen issues with the jets being too fast for the stream and strange things happen. 

As a note, I fly SEL and MEL for hours on end into different area's and never get crashes when doing so. I don't fly any airliners at all.

**Reply by rayw45** [reply]
Thanks for the feedback.  Well I will take back what I said as now I find if I just wait when X Plane freezes, after probably less than a minute, and often much less,  it will always come back without any loss of my flight situation.  If it greys out and I get the option to close or wait for it to respond, now I select wait and it comes back every time without crashing. Clearly this is trying to get the Bing sat image new tiles.

So what do you recommend I set the max wait to??  I am currently set to 5.0 so should I go back to the default 0.5 and would that help with the freezes and would I lose quality or not?  Mind you that is better than a freeze though.  Also you reckon with ZL=12 I will not lose any noticeable quality than 16, and even if flying low and slow ??  If left at ZL=16 will in time thatg result in higher res tiles on Bing ?? I do have Sim Heaven with the forests disabled as I use Global Forests V2.  Also I disable the auto ortho overlays as I uses the Sim Heaven overlays placed higher up the scenery_pack.ini file list.

Great piece of amazing software this truly is, I am stunned by the results I am now getting and be even greater when I can control those freezes too.   Just trying to get it optimised and then feel confident to help others having problems installing it and getting it all working smoothly..

**Comment by dpdurst** [comment]
Max wait is what you can tolerate, I have mine set for 1.5 on one PC and 0.5 on the other one. Again, I don't fly jets so I never notice a hiccup. With AO and either no Sim Heaven or with Sim Heaven I still get no pauses. ZL=12 you may not like but I've seen when steaming I get a mixture of 12 through 18 at times. I can tolerate any of the values flying low but then again I'm not really staring at the scenery either I'm flying. Might just want to test it and see what happens. I don't know if you'll ever get rid of the pauses with the Jets or at least that seems to be the consensus on the XP forums

**Reply by rayw45** [reply]
Thanks again dphurst. Yes the max wait set back to default 0.5 seems to make no difference.  I have a very fast full gigabit internet and good PC so I will try your setting of 1.5.  However it seems that I need to set that ZL down to default 12 and see the result.  I had not looked carefully at the setting in the config file as it is MINIMUM ZL not MAXIMUM as I wrongly thought.  I should have read that more carefully as now Min ZL = 12 is then not a problem.  Obviously the main thing is to set for no or absolute minimum pauses whilst flying as that is the priority of course, but I do like some of the scenery whilst flying in cruise as it adds to the suspension of disbelief.  

BTW how do you check what ZL level you are actually getting ??  Or is it just by eye guestimating ?? 

Like you I do not fly the big jets just yet but am loving the King Air 350 XP12 version which is a superb great plane to fly. Not fast like the airliners but is powerful and attains over 300kts but I fly mine about 260kts that is airspeed of course. I also have the Baron 58 and Cessna 172SP both with the excellent REPs. One thing I do not like with the C172SP is that flying manually there is no aileron or rudder trim (as is the case with the RW 172SP)  and despite my controllers (CH Flight Sim Yoke and Thrustmeter Rudder pedals with toe brakes) being correctly set up exactly to my liking, this plane refuses to fly straight hands off.  Maybe it is the hot turbulent air here in Thailand where I fly it.
